src/libAtomVM/context.c

```diff
@@ -309,6 +310,18 @@ bool context_get_process_info(Context *ctx, term *out, term atom_key)
             break;
         }
 
+        // registered_name for process or port..
+        case REGISTERED_NAME_ATOM: {
+            term name = globalcontext_get_registered_name_process(ctx->global, ctx->process_id);
+            if (term_is_nil(name)) {
```

Comment:
This code needs to be updated as well now, to check against term_is_invalid_term().